infra/examples-dev/aws-google-workspace/main.tf

```diff
@@ -35,11 +35,11 @@ provider "aws" {
 
 locals {
   base_config_path = "${var.psoxy_base_dir}/configs/"
-  bulk_sources = {
+  bulk_sources     = {
```

Comment:
fyi, I'm not sure this is how `terraform fmt` does it. I think we should follow whatever that does (and run `terraform fmt --recursive` to process whole tree)